azuread/resource_application.go

```diff
@@ -75,6 +76,16 @@ func resourceApplication() *schema.Resource {
 				Computed: true,
 			},
 
+			"group_membership_claims": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: suppress.CaseDifference,
+				ValidateFunc: validation.StringInSlice(
+					[]string{"None", "SecurityGroup", "All"},
+					true,
```

Comment:
Could we not ignore case here? We are slowly moving to treating as much as case sensitive as possible to get ahead of weird casing bugs we occasionally run into.